setup.cfg

```diff
@@ -0,0 +1,13 @@
+[check-manifest]
+ignore =
+    */test
+    */test/*
+    .*
+    node_modules
+
+[pep257]
+ignore = D202
+explain = true
+
+[yapf]
+based_on_style = pep8
```

Comment:
Do we use yapf at Hypothesis? I'm happy to adopt it if we want to but I dunno if I want to check in config for it if it's not part of our "official" toolchain